src/App.tsx

```diff
@@ -1,24 +1,184 @@
 import React from 'react';
 import './App.scss';
 
-const goodsFromServer: string[] = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
-
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
-
-export default App;
+type State = {
+  selectedGood: string;
+};
+
+export class App extends React.Component<{}, State> {
+  goodsFromServer = [
+    'Dumplings',
+    'Carrot',
+    'Eggs',
+    'Ice cream',
+    'Apple',
+    'Bread',
+    'Fish',
+    'Honey',
+    'Jam',
+    'Garlic',
+  ];
+
+  state = {
+    selectedGood: '',
```

Comment:
```suggestion
    selectedGoods: [],
```
store here every added good like this: `['carrot', 'fish', 'jam']`